package.json

```diff
@@ -64,6 +64,7 @@
     "moment": "2.10.6",
     "moment-timezone": "^0.4.0",
     "morgan": "1.2.0",
+    "ms": "0.7.1",
```

Comment:
I had raised initial concern about using [chrono.js](https://github.com/kkaefer/chrono.js), which I had actually confused with [chrono](https://github.com/wanasit/chrono), but the concern was valid anyway... they're both overkill for our needs to set a value in ms (e.g., chrono.js also has timezone support).

I'm not sure I see a lot of benefit to adding `ms` here because we only set a single value. Is the idea that we will start using `ms` everywhere that we set values in ms? If so, then I agree that it adds some readability and it's fine to add. But then we need to promote its use across calypso. Otherwise we're just adding overhead.